src/py/mains/run.py

```diff
@@ -87,6 +89,9 @@ def wait_for_file(filename, task, job_name = ""):
             if support.job_type == "SGE":
                 fc_run_logger.info( "deleting the job by `qdel` now..." )
                 os.system("qdel %s" % job_name) # Failure is ok.
+            if support.job_type == "SLURM"
+            	fc_run_logger.info( "Deleting the job by 'scancel' now...")
```

Comment:
Ahh, good to know. Thanks! Any chance you know if this extension exists for Chrome? I’ll also do a quick Google search as well.

I also plan on submitting a pull request to add support for PBS in the next 24 hours or so, so I’ll use all these recommendations for that code.

Roberto Lleras
Senior Scientist, Bfx. Field Applications Specialist, North America & Europe
Tel: +1.718.578.8987 | Skype: live:roberto.a.lleras_1 | rlleras@pacb.com<mailto:rlleras@pacb.com>
Pacific Biosciences | 1380 Willow Road, Menlo Park, California 94025 | pacb.com<http://www.pacificbiosciences.com/>

From: Christopher Dunn [mailto:notifications@github.com]
Sent: Monday, August 31, 2015 1:53 PM
To: PacificBiosciences/FALCON
Cc: Roberto Lleras
Subject: Re: [FALCON] Fix support for FALCON 0.3 running on SLURM (#181)


In src/py/mains/run.py<https://github.com/PacificBiosciences/FALCON/pull/181#discussion_r38338439>:

> @@ -87,6 +89,9 @@ def wait_for_file(filename, task, job_name = ""):

>              if support.job_type == "SGE":

>                  fc_run_logger.info( "deleting the job by `qdel` now..." )

>                  os.system("qdel %s" % job_name) # Failure is ok.

> +            if support.job_type == "SLURM"

> +             fc_run_logger.info( "Deleting the job by 'scancel' now...")

so I used the terrible in line text editor Git provides via the HTML website.

Interesting. GitHub usually gets this stuff right, and Ace is a pretty decent editor. Maybe it didn't detect the indentation because of mistakes elsewhere in the code. I'll strip them all soon.

  *   https://github.com/blog/905-edit-like-an-ace

For code which already has tabs, you can alter Ace rendering using the Stylish extension to Firefox with this Userstyle:

  *   https://userstyles.org/styles/70979/github-better-sized-tabs-in-code

—
Reply to this email directly or view it on GitHub<https://github.com/PacificBiosciences/FALCON/pull/181/files#r38338439>.